eidas_node/models.py

```diff
@@ -282,17 +288,17 @@ def deserialize_attributes(attributes_elm: Element) -> Dict[str, List[str]]:
     """Deserialize eIDAS attributes."""
     attributes = OrderedDict()  # type: Dict[str, List[str]]
     for attribute in attributes_elm:
-        if attribute.tag != 'attribute':
+        if attribute.tag.rpartition('}')[2] != 'attribute':
```

Comment:
Rather than this, the RHS should be constructed with the namespace.

Reply:
This approach is compliant with both 2.4 and 2.5 version. It validates in both cases when XML is without namespace (2.4) and with namespace (2.5)

Reply:
If I understand it correctly, then in 2.4 the namespace is empty and the result would be the same?

Reply:
Changed to QName(attribute.tag).localname